### ISSUE TITLE
`RenderableShadowCone` umbral rendering bug

### ISSUE THREAD

**Issue by WeirdRubberDuck** [OpenSpace/OpenSpace#3219]
When increasing the shadow length of a `RenderableShadowCone`, the umbral shadow stars increasing in size, when it should really disappear.

See image below where a shadow length of 2 is used for the Moon-Earth shadow.  Notice how the inner, turquoise shadow gets bigger to the left, when it really should be indefinitely thin

![image](https://github.com/OpenSpace/OpenSpace/assets/8808894/2eb287b8-026d-48db-86e2-b5fd31098279)

**Comment by alexanderbock** [comment]
I don't think that is correct?  The shadow would start forming the antumbra, see:  https://nineplanets.org/wp-content/uploads/2020/11/The-antumbra.jpg

The path of totality is basically the focal point of the moon's shadow, which means it would start diverging past the Earth again?

**Comment by WeirdRubberDuck** [comment]
> I don't think that is correct? The shadow would start forming the antumbra, see: https://nineplanets.org/wp-content/uploads/2020/11/The-antumbra.jpg
> 
> The path of totality is basically the focal point of the moon's shadow, which means it would start diverging past the Earth again?

Huh, I would expect it to disappear (i.e. be indefinitely thin). But that was just my intuition, so you are probably right :) 
Closing
